src/components/DeleteItemModal.jsx

```diff
@@ -0,0 +1,38 @@
+import Button from './Button';
+import ReactModal from 'react-modal';
```

Comment:
Great use of the react-modal library! I might utilize it in the future for my own project!

Reply:
I will def utilize this library!